TheMovieDb/Components/MainCoordinator.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  MainCoordinator.swift
+//  TheMovieDb
+//
+//  Created by Juan David Torres on 04/11/21.
+//
+
+import Foundation
+import UIKit
+final class MainCoordinator: Coordinator {
+  var children = [Coordinator]()
+  var navigationController: UINavigationController
+  
+  init(navigationController: UINavigationController) {
+    self.navigationController = navigationController
+  }
+  
+  func start() {
+    let vc = MainViewController()
+    vc.coordinator = self
+    navigationController.pushViewController(vc, animated: false)
+  }
+  
+  func showDetailMovie(movieTitle: String, movieScore: Float, posterPath: String, overview: String, id: Int) {
+    let vc = MovieDetailViewController(movieTitle: movieTitle, movieScore: movieScore, posterPath: posterPath, overview: overview, id: id)
```

Comment:
Can we pass a model for this arguments?

Reply:
also vc is kinda confusing, why not instead viewController, remember swift language is about provide clarity on the context

Reply:
Yes, the idea will be to use the viewmodel